libs/rest/src/Error.ts

```diff
@@ -5,8 +5,9 @@ import type { Response } from 'node-fetch';
 export const CordisRestError = makeCordisError(
   Error,
   {
-    retryLimitExceeded: (request: string, attempts: number) => `Tried to "${request}" for ${attempts} times but all of them failed.`,
-    requestTimeout: (request: string) => `Request "${request}" timed out.`
+    retryLimitExceeded: (attempts: number) => `Tried to make request ${attempts} times but all of them failed`,
```

Comment:
Is there a reason we're removing the path? Would be pretty useful for debugging to know..

Reply:
That was mostly there from my stack trace frustrations. If that's been fixed then the error nodejs throws should point to your actual function call so the path it hit is pretty obvious. We could keep it if you think it's still useful though.

Reply:
Hmmm, but isn't that stack trace gonna be a little muddled cause it's an async function? Recalling from d.js errors, the stack traces almost never pointed to the actual function calls, and this could be a hinderance for those using .then/.catch.

I would think for debugging/error handling purposes, incase people want to format how they are informed of ratelimits, providing the path is useful.

Reply:
> Hmmm, but isn't that stack trace gonna be a little muddled cause it's an async function? Recalling from d.js errors, the stack traces almost never pointed to the actual function calls

Yeah, I attempted to fix that with this PR, if it didn't, I'm trying again next patch.

As per the rest of your comment, fair point. Reverting that change.